content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,37 @@
+    
+- Metrics functionality included in the OpenTelemetry SDKs for Java, .Net, Python, and JS (coming next week) SDKs, which capture metrics from the APIs and perform some amount of processing. Metrics support for other languages is still in development. These are useful for:
+    - Developers of applications that are used by other organizations, like databases, message queues, etc., who will expose metrics over OTLP (or Prometheus) so that their own end users are able to monitor the performance of these apps. These applications could be open or closed source.
+    - Developers of applications within a technology organization, who want to capture metrics generated by OpenTelemetry APIs within their applications, either by their own developers or from shared libraries that their applications depend on. These metrics can be exported through OTLP or exported through any other OpenTelemetry exporter.
+    
+- Collector support for metrics, which includes the Collector’s ability to capture metrics from receivers for sources like host metrics or pre-packaged applications (these were already present in the Collector), the ability to receive metrics from protocols like OTLP and Prometheus, configuration-driven metric processing, and export these metrics to any supported destination. This is useful for:
+    - Anyone who wants to capture metrics from their hosts (Linux VMs, Windows, VMs, Kubernetes, etc.) or pre-packaged applications (databases, message queues, etc.).
+    - Anyone who wants to capture metrics from existing sources like OTLP (from OpenTelemetry SDKs, pre-packaged applications, etc.), Prometheus, or others.
+    - Anyone who wants to process / modify metrics and metric metadata captured from these sources.
+    - Anyone who wants to convert metrics from one format to another. For example, the Collector can capture metrics from a mix of OTLP and Prometheus sources, and then send all of these to a single destination using OTLP (with the standard OpenTelemetry semantic conventions), Prometheus, or any other exporter.
+    
+- Full OpenTelemetry Protocol (OTLP) support for efficiently serializing and transmitting metrics between systems.
+- A metrics section of the specification, which defines different types of metrics, their shapes, how to process them, and semantic conventions. This is primarily used by OpenTelemetry contributors but also provides guidance to OpenTelemetry users who are authoring metrics or metadata.
+
+All of this functionality is additive to OpenTelemetry’s existing tracing support, and both signal types share the same metadata and semantic conventions. As of this announcement, the Java, .Net, and Python SDKs have issued release candidates for OpenTelemetry’s metric capabilities, JS is coming next week, and more languages will be issuing metrics release candidates throughout the coming months. Each of these releases will be followed by general availability after we receive feedback from users.
+
+## Getting Started
```

Comment:
Nite: add a space between the heading and the text that follows (here and below).

Reply:
```suggestion
## Getting Started

```